test/map/transform.test.cpp

```diff
@@ -371,7 +371,7 @@ TEST(Transform, Antimeridian) {
     transform.jumpTo(CameraOptions().withCenter(coordinateWaikiri).withZoom(10.0));
     ScreenCoordinate pixelWaikiri = transform.latLngToScreenCoordinate(coordinateWaikiri);
     ASSERT_DOUBLE_EQ(500, pixelWaikiri.x);
-    ASSERT_DOUBLE_EQ(500, pixelWaikiri.y);
+    ASSERT_NEAR(500, pixelWaikiri.y, 1e-10);
```

Comment:
Just curious, why this is changed? Was there a precision loss?

Reply:
yes, indeed, there is a difference in calculated result related to precision loss - should have mentioned it.
Instead of 500, value 500.0000000000785 was calculated. Given the values, I find this difference acceptable.

Reply:
My assumption is that this code has landed in `easeTo` to conform with the `flyTo` implementation. This rounding error looks OK.